src/common/libflux/watcher_wrap.c

```diff
@@ -176,97 +192,97 @@ flux_watcher_t *flux_timer_watcher_create (flux_reactor_t *r,
                                            flux_watcher_f cb,
                                            void *arg)
 {
-    ev_timer *tw;
+    struct tmwatcher *tmw;
     flux_watcher_t *w;
     if (after < 0 || repeat < 0) {
         errno = EINVAL;
         return NULL;
     }
-    if (!(w = watcher_create (r, sizeof (*tw), &timer_watcher, cb, arg)))
+    if (!(w = watcher_create (r, sizeof (*tmw), &tmwatcher_ops, cb, arg)))
         return NULL;
-    tw = watcher_get_data (w);
-    ev_timer_init (tw, timer_cb, after, repeat);
-    tw->data = w;
+    tmw = watcher_get_data (w);
+    ev_timer_init (&tmw->evw, tmwatcher_cb, after, repeat);
+    tmw->evw.data = w;
 
     return w;
 }
 
 void flux_timer_watcher_reset (flux_watcher_t *w, double after, double repeat)
 {
-    assert (watcher_get_ops (w) == &timer_watcher);
-    ev_timer *tw = watcher_get_data (w);
-    ev_timer_set (tw, after, repeat);
+    assert (watcher_get_ops (w) == &tmwatcher_ops);
+    struct tmwatcher *tmw = watcher_get_data (w);
+    ev_timer_set (&tmw->evw, after, repeat);
 }
 
 void flux_timer_watcher_again (flux_watcher_t *w)
 {
-    assert (watcher_get_ops (w) == &timer_watcher);
+    assert (watcher_get_ops (w) == &tmwatcher_ops);
+    struct tmwatcher *tmw = watcher_get_data (w);
     struct ev_loop *loop = watcher_get_ev (w);
-    ev_timer *tw = watcher_get_data (w);
-    ev_timer_again (loop, tw);
+    ev_timer_again (loop, &tmw->evw);
 }
 
 /* Periodic
  */
-struct f_periodic {
+
+struct pdwatcher {
     struct flux_watcher *w;
-    ev_periodic          evp;
-    flux_reschedule_f    reschedule_cb;
+    ev_periodic evw;
+    flux_reschedule_f reschedule_cb;
 };
 
-static void periodic_start (flux_watcher_t *w)
+static void pdwatcher_start (flux_watcher_t *w)
```

Comment:
Just a comment, I don't think this quite rises to the level of requiring a change to whole PR:

Was there a reason `NAME` for these watchers was kept to only 2 characters? I'm finding `pdwatcher` almost more unclear than `periodic_start`. Could it have have `struct periodic_watcher` and `periodic_watcher_start`?

(Same with `tm` vs `timer`)

Reply:
Sorry, that actually would be better.  I'm not sure why I was stingy with the characters there.  Old habits I suppose.